app/src/main/java/org/fossasia/pslab/activity/OscilloscopeActivity.java

```diff
@@ -252,6 +252,9 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             case "Speed of Sound":
                 isSpeedOfSoundExperiment = true;
                 break;
+            case "WaveGenerator":
+                //do nothing
+                break;
```

Comment:
What's this one doing here?

Reply:
without this the oscilloscope is crashing